SwaprHQ/swapr-dapp
# feat: add stackly popup

## Fixes: #swa-62

# Description

This PR adds stackly popup in yellow and green type. Advise to test it on preview link (or build it locally) then to test it locally, will fail to show the modal due to how react works locally.

To make it work don't forget to set env variables:

`REACT_APP_STACKLY_POPUP_COLOR=yellow`
`REACT_APP_SHOW_STACKLY_POPUP=true`

![image](https://github.com/SwaprHQ/swapr-dapp/assets/23079677/bf61bde5-1f11-4cc8-9b9e-e8aee75c6278)

# How to test the changes

I advise to test it in the preview. Once you close the only way to see it again is through cleaning local storage.
